docs/release-notes/cloud/cloud-2024-02.md

```diff
@@ -0,0 +1,91 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud February 2024.
+included_jira_versions: ['2024.02']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.193
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.234
+---
+
+# Cloud February 2024
+
+These release notes are for the Codacy Cloud updates during February 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The Codacy GitHub App no longer requires **Read and write access to Administration** repository permission. For more information, see the [Discontinuation of SSH keys for GitHub repositories](./cloud-2024-01-15-gh-repository-ssh-keys-discontinuation.md). (PLUTO-803)
+-   The coverage data sent by the new Coverage engine to your Git provider moved from **beta** to stable. Thus, the coverage data sent by the old engine is now marked **deprecated**. For more information, [see the rollout details of the new Coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md). (ALA-766)
+-   The [Coverage Pull Requests page](../../repositories-coverage/pull-requests.md) now gets data from the new Coverage engine. For more information, [see the rollout details of the new Coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md). (ALA-770)
+-   The [Organization overview](../../organizations/organization-overview.md) is now available for the open source plan. (ALA-842)
```

Comment:
Should we mention "on all plans" as an inclusive version? Or could it confuse some users?

```suggestion
-   The [Organization overview](../../organizations/organization-overview.md) is now available on all plans. (ALA-842)
```